File: CodeWars/Even Or Odd.java
// Create a function (or write a script in Shell) that takes an integer as an 
// argument and returns "Even" for even numbers or "Odd" for odd numbers.

public class EvenOrOdd {
    public static String even_or_odd(int number) {
      int  res;
      String o = "Odd";
      String e = "Even";
      
      res = number % 2;
      if (res != 0)
        return o;
      return e;
        
    }
}

File: CodeWars/Even or Odd.m
// Create a function (or write a script in Shell) that takes an integer as an 
// argument and returns "Even" for even numbers or "Odd" for odd numbers.

#import <Foundation/Foundation.h>
NSString* evenOrOdd (NSInteger n){
  NSString *even = @"Even";
  NSString *odd = @"Odd";
  if ((n % 2) == 0)
    return even;
  else
    return odd;
}